tree/ntuple/inc/ROOT/REntry.hxx

```diff
@@ -177,12 +182,22 @@ public:
       fValues[token.fIndex].Bind(objPtr);
    }
 
+   /// Bind the value for the field, referenced by its name, to `objPtr`.
+   ///
+   /// Ownership is shared with the caller and the object will be kept alive until it is replaced (by a call to
+   /// EmplaceNewValue, BindValue, or BindRawPtr) or the entry is destructed.
+   ///
+   /// **Note**: if `T = void`, type checks are disabled. It is the caller's responsibility to match the field and
+   /// object types.
```

Comment:
I am not sure but maybe either
```suggestion
   /// object's type.
```
or
```suggestion
   /// objects' type.
```

Reply:
I thought about this and my feeling (not a native speaker) was that "field and object types" conveys the message the best. See for example also https://www.w3schools.com/typescript/typescript_object_types.php for the term "object type".

Reply:
Pretty sure the original formulation is ok.